Sources/PitchSpeller/Wetherfield/EdgeCapacity.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  EdgeCapacity.swift
+//  NotationModel
+//
+//  Created by Ben Wetherfield on 06/16/18.
+//
+
+public enum EdgeCapacity:
+    Comparable,
+    Equatable,
+    Hashable
+{
+    case real(UInt)
+    case bigM
+    
+    public var hashValue: Int {
```

Comment:
Ditto re: `Hashable`. `struct` and `enum` values whose comprising values are `Hashable` get it for free.